cves/CVE-2014-0473.yml

```diff
@@ -60,68 +74,28 @@ bounty:
   url:
 reviews: []
 bugs: []
-repo:
+repo: https://github.com/django/django
 fixes_vcc_instructions: |
   Please put the commit hash in "commit" below (see my example in
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
 - commit: 1170f285ddd6a94a65f911a27788ba49ca08c0b0
-  note:
+  note: fix for Django version 1.4 added a check in cache middleware to prevent caching responses that set a user-specific cookie in response to a cookie-less request
 - commit: 6872f42757d7ef6a97e0b6ec5db4d2615d8a2bd8
-  note:
+  note: fix for Django version 1.5
 - commit: d63e20942f3024f24cb8cd85a49461ba8a9b6736
-  note:
+  note: fix for Django version 1.6
 - commit: 380545bf85cbf17fc698d136815b7691f8d023ca
-  note:
+  note: fix for Django version 1.7
 vccs:
-- commit: d65526d6886067a8ef368e5b02fce80e1e4c4903
-  note: This VCC was discovered automatically via archeogit.
 - commit: 752659037686026e85c2da52c756d4bfec8c4d13
   note: This VCC was discovered automatically via archeogit.
 - commit: 00fda7f45db2425c1dcb5d927093ede45734d841
   note: This VCC was discovered automatically via archeogit.
-- commit: 7cb140e6d86e9c592983ddb50bfd82480ecec7eb
-  note: This VCC was discovered automatically via archeogit.
-- commit: fe581013b03b3dd38f4893bb5e9dc9695af25d71
-  note: This VCC was discovered automatically via archeogit.
 - commit: ed7a30782b7bf0e30fd47fc407e3b9c22bd80d2f
   note: This VCC was discovered automatically via archeogit.
-- commit: 673e6fc7fb243ed44841b9969d26a161c25733b3
-  note: This VCC was discovered automatically via archeogit.
-- commit: d0eb4693ab479ef0667cd5286ecc89c347b6b2de
-  note: This VCC was discovered automatically via archeogit.
-- commit: f6c991667f6f1ab8d2fd2afc1ee2b5801e78b7e3
-  note: This VCC was discovered automatically via archeogit.
-- commit: 4819797f70ce718276466962418ba5ecc80d0fba
-  note: This VCC was discovered automatically via archeogit.
-- commit: afd040d4d3a06fe92e3080870b2ff2095ce86a75
-  note: This VCC was discovered automatically via archeogit.
-- commit: cd914175c8209c5f366e87d94f8f6d050347757d
-  note: This VCC was discovered automatically via archeogit.
-- commit: 4b27813198ae31892f1159d437e492f7745761a0
-  note: This VCC was discovered automatically via archeogit.
-- commit: 292322f977b1844e15ba25d59d1e985461571c15
-  note: This VCC was discovered automatically via archeogit.
-- commit: faf570df187bc82a3bcffa5613165a6e6cc56a57
-  note: This VCC was discovered automatically via archeogit.
-- commit: 24de85c419ed46baf95098822f3f455887bc00f6
-  note: This VCC was discovered automatically via archeogit.
-- commit: 1c8be95a864540d416602577d1aa03d58ba33168
-  note: This VCC was discovered automatically via archeogit.
 - commit: b22415214a7bdeaf8ccd7b8b21872038ab865991
   note: This VCC was discovered automatically via archeogit.
```

Comment:
are there any significant parts of this commit that led to the vulnerability you want to comment on? Did they touch this code area where the problem is, did they make things worse, etc?